rust/agama-lib/src/network/proxies.rs

```diff
@@ -21,9 +21,9 @@
 //! D-Bus interface proxies for: `org.opensuse.Agama*.**.*`
 //!
 //! This code was generated by `zbus-xmlgen` `3.1.0` from DBus introspection data.`.
```

Comment:
NP: I am not sure if we should keep this as it is confusing...I hit it often in past that I have newer zbus_xmlgen installed and result is wrong and need adaptation.

Reply:
That's the intention of separating many of those files into smaller ones, to autogenerate them someday (not yet). In any case, that message is wrong, as we are using version 5 now.

Reply:
Actually this one was generated with that version. I will update all those proxies (which are working) on a separate PR if needed.